src/gui/fxml/Home.fxml

```diff
@@ -15,13 +15,13 @@
         <items>
           <AnchorPane minHeight="0.0" minWidth="0.0">
                <children>
-                  <Button mnemonicParsing="false" onAction="#addFolder" text="+" AnchorPane.rightAnchor="20.0" AnchorPane.topAnchor="10.0" />
+                  <Button mnemonicParsing="false" onAction="#addFolder" text="%home_folderAdd" AnchorPane.rightAnchor="20.0" AnchorPane.topAnchor="10.0" />
```

Comment:
Let's use snake case for every string property, change `home_folderAdd` to `home_folder_add`